tests/booted/README.md

```diff
@@ -0,0 +1,4 @@
+# Booted tests
+
+These are intended to run via tmt; use e.g.
+`make test-tmt`.
```

Comment:
might want to add a note that this requires a rootful connection to podman, e.g. `export CONTAINER_CONNECTION="podman-machine-default-root"`. Also might be good to note this will appear to hang while copying the bootc source into the container since the source directory is large.

Reply:
> Also might be good to note this will appear to hang while copying the bootc source into the container since the source directory is large.

See https://github.com/teemtee/tmt/pull/3037